scripts/gulpfiles/build_tasks.js

```diff
@@ -231,6 +229,8 @@ const JSCOMP_ERROR = [
  * it's generally sufficient to remove them from JSCOMP_ERROR.
  */
 const JSCOMP_WARNING = [
+  'deprecated',
+  'deprecatedAnnotations',
```

Comment:
This is going to lead to seeing warnings during build.  It that what we want?

More broadly, why do we have non-deprecated code calling `@deprecated` code?

Reply:
> More broadly, why do we have non-deprecated code calling @deprecated code?

We don't, in TypeScript. Beka made this a warning because it was previously an error. The line that Closure complains about is actually the line that declares the enum. The generated JS looks different from the TS, and causes this error. 

Adding a `@suppress` directly into the TsDoc already present on the enum did not cause closure to suppress the error either, but I don't know why. The only way we could get it to suppress the error was to add a new jsdoc comment containing only the `@suppress`, but that breaks the tsdoc generated reference docs as the real comment is no longer associated with the enum declaration. Would be interested in your thoughts on this problem, but in the meantime I suggested that calling deprecated code shouldn't be build-breaking IMO, but a warning seems appropriate.